ApiCheck.NUnit/ApiTest.cs

```diff
@@ -47,11 +41,11 @@ private IEnumerable<ApiTestData> GetResults()
         {
           ApiTestAttribute apiTestAttribute = (ApiTestAttribute)customAttribute;
 
-          Stream comparerConfigurationStream = GetReadStream(apiTestAttribute.ComparerConfigurationPath);
+          Stream comparerConfigurationStream = GetReadStream(Path.GetFullPath(apiTestAttribute.ComparerConfigurationPath, Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)));
```

Comment:
Why does the relative path not work anymore? And this conversion from relative to absolute path is duplicated in several places. Can be moved to some utility class.

Reply:
If I remember correctly the execution context was different than the original code so the current directory path is different now. This was breaking the tests.

Reply:
I removed unneeded path conversions.